### PR TITLE
Draft release notes up to brim/1738

### PR DESCRIPTION
**Pull request by philrz**
This is an exercise equivalent to the one described for Zed in brimdata/zed#2872. See the description of that PR for background.

(The failure of the markdown link checker is due to my having linked to the expected next Zed release version, which, indeed, does not yet exist. Drafts be draftin'.)

### CODE REVIEW

**Review by nwt — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,22 +14,34 @@
...
 * Fix an issue where viewing Suricata alerts in Brim could trigger a "TypeError: Cannot read property 'startsWith' of undefined" error (#1706)
-* Ensure Pool updates made from outside the app are reflected automatically in Brim (#1702, #1709, #1711, #1713, #1722, #1733, #1734)
-* Update the [Brim wiki](https://github.com/brimdata/brim/wiki) articles for the `v0.25.0` release, including separate articles for old & new Brim where functionality has changed significantly (#1723)
+* Ensure pool updates made from outside the app are reflected automatically in Brim (#1702, #1709, #1711, #1713, #1722, #1733, #1734)
+* The [Brim wiki](https://github.com/brimdata/brim/wiki) now has articles for changes specific to the `v0.25.0` release, with parallel articles remaining for older Brim where functionality has changed significantly (#1723)
 * Update the [Code Base Walkthrough wiki article](https://github.com/brimdata/brim/wiki/Code-Base-Walkthrough) to reflect recent changes to where code lives in the repo (#1738)
+* **Preferences** now has settings for a "thousands separator" and "decimal" that allow changing from `,` & `.` defaults or removing them entirely (#1740, #1765)
```
Comment (nwt) [inline CHANGELOG.md:33]:
```suggestion
* **Preferences** now has settings for a "thousands separator" and "decimal" that allow changing from `,` and `.` defaults or removing them entirely (#1740, #1765)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,22 +14,34 @@
...
+* Fix issues where error messages from failed imports were not being fully surfaced (#1760, #1786)
+* Zed `type` values that are output in Brim are no longer wrapped with parentheses (#1757)
+* Hovering over a field value now displays the Zed data type in a tooltip (#1766)
+* Fix an issue where selecting a range from the time span pull-down caused a crash with unshaped data (#1777)
+* On macOS, if all Brim windows are closed but the app is still running, clicking the Brim icon in the dock now opens a new window (#1782)
```
Comment (nwt) [inline CHANGELOG.md:38]:
```suggestion
* On macOS, if all Brim windows are closed but the app is still running, clicking the Brim icon in the Dock now opens a new window (#1782)
```

**Review by nwt — APPROVED** [review]

**Review by jameskerr — APPROVED** [review]
That's a lot of fixes! 

Just an idea. Some projects I follow have sections for "Features" and sections for "Bug Fixes". Might be nice in the future to try to make a distinction in the release notes.

**Comment by philrz** [comment]
Thanks to everyone who gave this a look! I'm going to close this draft PR and open a new official one during the GA release process I'm now starting.

**Pull request closed without merging**